service/dap/server.go

```diff
@@ -918,9 +1035,12 @@ func (s *Server) stopNoDebugProcess() {
 }
 
 // TODO(polina): support "remote" mode
+// TODO(polina): document behavior for remaining modes
```

Comment:
Did you mean for this to be a TODO on polina or yourself?

Reply:
It was added on my behalf by polina's request

Reply:
https://github.com/go-delve/delve/pull/2575